generator.rb

```diff
@@ -0,0 +1,66 @@
+def is_vowel?(letter)
+  if (letter == "a") || (letter == "e") || (letter == "i") || (letter == "o") || (letter == "u")
+    return true
+  else
+    return false
+  end
+end
+
+def run_generator
+  # Prompts user for the first word
+  puts "What is your first word to combine?"
+  word_a = gets.chomp
+  until word_a.length > 1
+    puts "Invalid input, must be at least 2 characters long."
+    puts "Please re-enter the input."
+    word_a = gets.chomp
+  end
+
+  # Prompts user for the 2nd word
+  puts "What is your 2nd word to combine?"
+  word_b = gets.chomp
+  until word_b.length > 1
+    puts "Invalid input, must be at least 2 characters long."
+    puts "Please re-enter the input."
+    word_b = gets.chomp
+  end
```

Comment:
our code to get input and validate it looks pretty similar between `word_a` and `word_b`... In the future, this is a good chance to refactor :)